alf/environments/gym_wrappers.py

```diff
@@ -129,7 +129,8 @@ def transform_space(self, observation_space):
                 low = observation_space.low
                 high = observation_space.high
                 if np.isscalar(low) and np.isscalar(high):
-                    shape = observation_space.shape[::-1]
+                    shape = (observation_space.shape[-1:] +
```

Comment:
Does this suggest the previous implementation is a bug that happen to work because most images are square?

Reply:
> Does this suggest the previous implementation is a bug that happen to work because most images are square?

Right the current one is buggy, only if low and high are scalars and width and height are different.